libraries/octree/src/OctreeConstants.h

```diff
@@ -37,7 +37,7 @@ const int NUMBER_OF_CHILDREN = 8;
 
 const int MAX_TREE_SLICE_BYTES = 26;
 
-const float VIEW_FRUSTUM_FOV_OVERSEND = 60.0f;
+const float VIEW_FRUSTUM_FOV_OVERSEND = 0.0f;
```

Comment:
This hackery is no longer helpful for the entity-server, at least not when testing correctness.  We may want to re-tune some non-zero OVERSEND later once we know things are working correctly.  This may also affect avatar-mixer if it uses the same viewFrustum to figure out who is in view -- again, lets' test and find out.